x-pack/filebeat/module/rabbitmq/log/ingest/pipeline.yml

```diff
@@ -0,0 +1,27 @@
+---
+description: Pipeline for parsing RabbitMQ logs
+processors:
+- grok:
+    field: message
+    pattern_definitions:
+      GREEDYMULTILINE: "(.|\n)*"
+      ERL_PID: "\\<%{INT}+\\.%{INT}+\\.%{INT}+\\>"
+    patterns:
+    - "%{DATESTAMP:timestamp} \\[%{WORD:log.level}\\] %{ERL_PID:rabbitmq.log.pid}
+      %{GREEDYMULTILINE:message}"
+    ignore_missing: true
+- date:
+    field: timestamp
```

Comment:
I somehow miss the part where it's converted to `@timestamp` but based on the output below it works. I'm confused.

Reply:
`field` is the input field. `target_field` is the output field and [it defaults to `@timestamp`](https://www.elastic.co/guide/en/elasticsearch/reference/master/date-processor.html#date-processor)

I omitted it because I want the default behavior, but I can make it explicit if that's preferable.

Reply:
Good to know, somehow missed that default. I'm on the fence on this one. I really like that `@timestamp` is the default because of ECS. At the same time having it in here would make it more obvious what is happening for an other engineering touching this. I'm good with both.